pkg/crane/pull.go

```diff
@@ -63,7 +63,8 @@ func Save(img v1.Image, src, path string) error {
 		tag = d.Repository.Tag(iWasADigestTag)
 	}
 
-	return tarball.WriteToFile(path, tag, img)
+	// no progress channel (for now)
+	return tarball.WriteToFile(path, tag, img, nil)
```

Comment:
You can just drop the final nil here.

Reply:
In fact, this nil here causes a panic in travis

Reply:
Yes, good point. I will update it.

> this nil here causes a panic 

So, technically, that is a "nil panic", which means "no panic", so there isn't a panic? :-)